src/Ball2.jl

```diff
@@ -26,7 +26,7 @@ We evaluate the support vector in a given direction:
 
 ```julia
 julia> σ(ones(5), B)
-5-element Array{Float64,1}:
+5-element Vector{Float64}:
```

Comment:
well if this was a doctest (instead of putting `julia`, write `@jldoctest` as explained in Documenter.jl, then it should be exactly the same as in julia REPL's output, i guess.. i haven't checked, though, we can leave it like that and see when we add it as a doctest.

Reply:
Ah, now I understand your point here.
I agree, this will probably break.
I fixed that directly in master.